app/include/zmk/keymap.h

```diff
@@ -6,7 +6,10 @@
 
 #pragma once
 
+u8_t zmk_layer_default();
+u32_t zmk_layer_state();
```

Comment:
When exposing bitflag values like this, I usually like to create a type alias to make them easier to work with remember what they are, etc, e..g

```
typedef u32_t zmk_layers_state;
```

or somesuch, and then use that pervasively.